src/main/java/frc/robot/RobotContainer.java

```diff
@@ -90,6 +92,29 @@ public RobotContainer() {
         configureSubsystems();
         configureModes();
         configureAutonomous();
+
+        NoteManager.addNote(new Note(driveTelemetry::getFieldToRobot, true, "1"));
```

Comment:
This may already be the case, but can we make sure this is only active in Sim mode?